index.js

```diff
@@ -0,0 +1,50 @@
+const express = require("express");
+const app = express();
+const axios = require("axios");
+const pool = require("./database");
+require("dotenv").config();
+
+app.use(express.json());
+
+let port = process.env.PORT || 3000;
+
+app.listen(port, () => {
+  console.log(`running on ${port}`);
+});
+
+app.post("/github_push_webhook/:id", async (req, res) => {
+  try {
+    const { id } = req.params;
+    var reqData = req.body;
+
+    const commit_message = reqData.head_commit.message || reqData.repository.id;
```

Comment:
this isn't making sense to me - can you find out why github is sending null commit message? it shouldn't be ideally and if it is, there should be some explanation for it